kong/plugins/request-transformer/access.lua

```diff
@@ -81,6 +81,15 @@ local function append_value(current_value, value)
 end
 
 local function transform_headers(conf)
+  -- Rename headers(s)
+  for _, name, value in iter(conf.rename.headers) do
```

Comment:
for those iterators, I think it would be clearer to use `_, old_name, new_name` instead of `_, name, value`, it makes the code easier to read.